ssh/ssh_sed_unlock.sh

```diff
@@ -0,0 +1,70 @@
+#!/usr/bin/env bash
+
+trap '' 2
+
+echo
+while [ true ] ; do
+
+    if /usr/local/sbin/sedunlocksrv/opal-functions.sh | grep -q "Could not detect TCG Opal 2-compliant disks"
+    then
+        echo "Could not detect TCG Opal 2-compliant disks. Nothing to do here."
+        echo
+        echo "Press [ENTER] to shutdown"
+        while [ true ] ; do
+            read -s -N 1 -t 1 key
+            if [[ $key == $'\x0a' ]];        # if input == ENTER key
+            then
+                echo "Shutting down..."
+                poweroff -nf
+            fi
+            done
+    fi
+
+    echo -n "🔑🔑🔑 Enter SED password: "
+
+    unset password;
+    while IFS= read -r -n1 -s char; do
+        case "$char" in
+        $'\0')
+            break
+            ;;
+        $'\177')
```

Comment:
Is this the backspace? A comment would be handy :)

Reply:
yes it is. Will add comments.

Reply:
Thanks!

Reply:
done